### PR TITLE
cleanup: Make pylint and mypy happy with bootstrap_node_info.py.

### PR DESCRIPTION
**Pull request by iphydf**
This change is [Reviewable](https://reviewable.io/reviews/toktok/c-toxcore/1516)

### CODE REVIEW

File: other/fun/bootstrap_node_info.py [hunk 1]
```diff
@@ -42,57 +43,78 @@ def printHelp():
...
+              "is either offline or has no info set.")
+        sys.exit(2)
+
+    packet_id = data[:PACKET_ID_LENGTH]
+    if packet_id != INFO_PACKET_ID:
+        print("Bad response, first byte should be " + str(INFO_PACKET_ID) +
+              " but got {packet_id}({data})".format({
+                  "packet_id": packet_id,
+                  "data": data,
+              }))
```
Comment (nurupo) [inline other/fun/bootstrap_node_info.py:97]: `format()` [doesn't take a dict](https://docs.python.org/3/library/stdtypes.html#str.format).

```
$ yes | nc -lvu 127.0.0.1 33445 > /dev/null 2>&1 &
[1] 1644781
$ ./bootstrap_node_info.py
Traceback (most recent call last):
  File "bootstrap_node_info.py", line 116, in <module>
    main(
  File "bootstrap_node_info.py", line 94, in main
    " but got {packet_id}({data})".format({
KeyError: 'packet_id'
[1]+  Done                    yes | nc -lvu 127.0.0.1 33445 > /dev/null 2>&1
```

Either change it to `format_map()` or use named arguments.

**Comment by nurupo** [comment]
>Make pylint and mypy happy with bootstrap_node_info.py

```
$ python3 -m mypy bootstrap_node_info.py
bootstrap_node_info.py:94: error: Cannot find replacement for named format specifier "packet_id"
bootstrap_node_info.py:94: error: Cannot find replacement for named format specifier "data"
bootstrap_node_info.py:94: error: Not all arguments converted during string formatting
Found 3 errors in 1 file (checked 1 source file)
```

Huh.

File: other/fun/bootstrap_node_info.py (hunk 1 above)
Comment (iphydf) [reply other/fun/bootstrap_node_info.py:97]: Done.

**Comment by iphydf** [comment]
I've looked into it, but I didn't want to do the python3 migration for that one at that point. I'll look into it later (unless you beat me to it).

**Pull request merged**